src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
+
+        if (doughnutType.equals("Chocolate")) {
```

Comment:
repetitive code, consider using a[ switch statement](https://docs.oracle.com/en/java/javase/17/language/switch-expressions-and-statements.html)